### PR TITLE
Add discuss.k8s.io to support response blurb

### PR DESCRIPTION
**Pull request by nikhita**
Would be useful to point them to discuss as well.

/assign @mrbobbytables @castrojo

### CODE REVIEW

File: contributors/guide/issue-triage.md [hunk 1]
```diff
@@ -71,8 +71,8 @@ If you see support questions on kubernetes-dev@googlegroups.com or issues asking
 support try to redirect them to Stack Overflow. Example response:
 
 ```code
-Please re-post your question to [Stack Overflow]
-(http://stackoverflow.com/questions/tagged/kubernetes).
+Please re-post your question to [Stack Overflow](http://stackoverflow.com/questions/tagged/kubernetes)
```
Comment (nikhita) [inline contributors/guide/issue-triage.md]: This moves the SO link one line up, because the preview looks awkward if copy pasted directly.

**Comment by mrbobbytables** [comment]
This might be better to discuss on the mailing list -- but I've been thinking about this. Do we want to continue to direct people to SO? Discuss has built up a pretty good user-base and SO is no longer an officially sponsored avenue (@coderanger noticed it back in January). I think directing folk to a single place for end-user related questions provides a better experience and has the added benefit of a platform we can integrate with and control.

 A good example might be [GitHub to Discourse Issue mover](https://meta.discourse.org/t/introducing-github-issues-to-discourse/46671). Issues that are support focused could be closed and moved over with a TBD bot command. 

That's just one possible example, I know @castrojo has thought of some other ideas :)

**Comment by mrbobbytables** [comment]
> We can talk SO separate from this PR, in the meantime we can merge this.

👍 

Agree. I wouldn't consider it a blocker, just figure it's a good time to kick up that discussion.

I don't have approve rights, @castrojo do you want to toss in an approve as well?

**Pull request merged**